thorlcr/activities/indexwrite/thindexwrite.cpp

```diff
@@ -298,7 +298,7 @@ class IndexWriteActivityMaster : public CMasterActivity
                 CDateTime modifiedTime(mb);
                 IPropertyTree &props = partDesc->queryProperties();
                 props.setPropInt64("@size", size);
-                StringBuffer dateStr;
+                props.setPropInt64("@recordCount", r);                StringBuffer dateStr;
```

Comment:
Should this be in the 0 == slaveIdx conddition or for all parts ?

Reply:
yes needs to be here, and not done if a single part key and slaveIdx>0  (same as other part props)

Reply:
Is there a newline missing ?